js/src/components/tree-select-control/checkbox.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { Icon, check } from '@wordpress/icons';
+
```

Comment:
💅  Add the unresolved `Option` type, or the `InnerOption` type if took this [suggestion](to_be_added).

```js
/**
 * @typedef {import('./').Option} Option
 */
```